PULL_REQUEST_TEMPLATE.md

```diff
@@ -28,3 +30,25 @@ We must be able to understand the design of your change from this description. I
 ### Applicable Issues
 
 <!-- Enter any applicable Issues here -->
+
+### Metrics
+
+<!-- What metrics are associated with this code change and what questions can they help us answer? -->
+
+### Tests
+
+<!-- What tests were added / will be added to help protect against future regressions? -->
+
+### Documentation
+
+<!-- Describe the documentation added or improved -->
+
+### Pre-Release Validation
```

Comment:
- If I was a community contributor, I might be confused by this question.

- I don't think we have hypotheses to answer for every pull request.  And if we collect those hypotheses on pull requests, that doesn't make it easy to aggregate them.  Instead, I think it makes the most sense to list our hypotheses in a release checklist, since that's the cadence we're aiming for with our research efforts.